Bio/MarkovModel.py

```diff
@@ -347,10 +350,7 @@ def _baum_welch_one(N, M, outputs,
 
 
 def _forward(N, T, lp_initial, lp_transition, lp_emission, outputs):
-    # Implement the forward algorithm.  This actually calculates a
-    # Nx(T+1) matrix, where the last column is the total probability
-    # of the output.
-
+    """Implement forward algorithm (PRIVATE)."""
```

Comment:
I think the original comment had useful information worth including as the main text of the new docstring.